src/Sparrow/LowMemory/DirtyMemoryState.cs

```diff
@@ -5,5 +5,7 @@ public class DirtyMemoryState
         public bool IsHighDirty;
 
         public long TotalDirtyInBytes;
+
+        public long TotalDirtyInMb => new Size(TotalDirtyInBytes, SizeUnit.Bytes).GetValue(SizeUnit.Megabytes);
```

Comment:
Move this to 4.2 and do the conversion inside the SNMP endpoint

Reply:
done